### ISSUE TITLE
BraTs Data

### ISSUE THREAD

**Issue by 6018203135** [Amshaker/unetr_plus_plus#37]
BraTs dataset in trainer train has 387 datasets, val has 73 datasets together a total of 460, but a total of 484, the author provides preprocessed imagesTs dataset is 485 after no labels, if not run directly run_evaluation_tumor.sh, run separately predict_simple.py At this time, pass in the parameter i What should the imagesTs path dataset be? Can you give a specific catalog training set, validation set, and test set? Or can I put all the data except for the test set (i.e. the val file of the source trainer) back under the trainer?

**Comment by Amshaker** [comment]
Hi @6018203135 ,

Please refer to our paper for the full details. We mentioned that we divided BraTs with 80:5:15 ratios for training, validation, and testing. This means that we have 387 training samples, 24 samples for internal validation (hyper-parameter tuning, not provided in the pre-processed dataset), and 73 samples for testing (which you call validation). You can follow the same folder structure of the pre-processed dataset without change.

I hope it is clear now.
Best regards,
Abdelrahman.

**Comment by 6018203135** [comment]
@Amshaker 
Thank you veay much!
